manim/_config/logger_utils.py

```diff
@@ -9,18 +9,22 @@
 format.
 
 """
-
-import os
-import logging
-import json
+import configparser
 import copy
+import json
+import logging
+import os
+import typing
+from typing import TYPE_CHECKING
 
+from rich import color, errors
+from rich import print as printf
 from rich.console import Console
 from rich.logging import RichHandler
 from rich.theme import Theme
-from rich import print as printf
-from rich import errors, color
 
+if TYPE_CHECKING:
+    from manim._config.utils import ManimConfig
```

Comment:
In someday future, we would have type checking as a test. At that time it will be super helpful. For now, my ide didn't detect it properly so I had this import statement. Also, seeing docs it just works. That if condition will never be True until type checking like mypy is used.